deploy/helm-chart/kubernetes-replicator/templates/rbac.yaml

```
@@ -20,9 +20,17 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "namespaces" ]
     verbs: [ "get", "watch", "list" ]
-  - apiGroups: [""] # "" indicates the core API group
-    resources: ["secrets", "configmaps"]
-    verbs: ["get", "watch", "list", "create", "update", "patch", "delete"]
+{{- range $index, $val := .Values.serviceAccount.privileges }}
+  - apiGroups:
+    {{- range $val.apiGroups }}
+      - {{ . | quote }}
+    {{- end }}
```

Also, maybe easier & shorter (untested, but should hopefully work):

```suggestion
  - apiGroups: {{ $val.apiGroups | toYaml | nindent 6 }}
```